pages/common/keychain.md

```diff
@@ -0,0 +1,28 @@
+# keychain
+
+> re-use ssh-agent and/or gpg-agent between logins
+> More information: <http://funtoo.org/Keychain>.
+
+- Check for a running ssh-agent, and start one if needed:
+
+`keychain`
+
+- Also check for gpg-agent:
+
+`keychain --agents "gpg,ssh"`
+
+- List signatures of all active keys and exit:
+
+`keychain --list`
+
+- List fingerprints of all active keys and exit:
+
+`keychain --list-fp`
+
+- Add a timeout for identities added to the agent, in minutes:
+
+`keychain --timeout {{minutes}}`
+
+- Enforce using gpg2 over gpg (for distributions like Ubuntu that have both)
```

Comment:
```suggestion
- Use GPG2 rather than GPG
```

Reply:
I would be open to just dropping this from the page if you think it's too high-level of a distinction for tldr.

Reply:
On my machine, GPG is a symbolic link to GPG2.

I don't know whether the command would be fine using the default method (without the --gpg2 flag), lets see what others think.

Reply:
Are you using ubuntu? It's possible that the manpage for keychain is just out of date, and this feature is obsolete.

Reply:
Arch Linux

Reply:
I'd assume that the only version of gpg used today is gpg2. I haven't heard of the original gpg1 being actually used at all?

Reply:
ok